lib/aws/broker/config.rb

```diff
@@ -0,0 +1,16 @@
+module Aws
+  class Broker
+    class Config
+
+      attr_accessor :enabled, :queue_prefix
+      attr_accessor :aws_access_key, :aws_secret_key, :aws_region
+      alias_method :enabled?, :enabled
```

Comment:
What is the purpose of having the enabled boolean?

Reply:
useful for doing something like
```
Broker.configure do |config|
  config.enabled = Rails.env.production?
end
```
otherwise it will trigger calls to AWS in dev

Reply:
Gotcha, I think we should figure out our environments better rather then relying on just `Rails.env.production?` for instance this will create queues in staging, not sure if that is what we want, how they will be scoped etc. This will work for now though.